pkg/api/routes.go

```diff
@@ -12,8 +12,8 @@ func (h Handler) initRoutes() *mux.Router {
 
 	h.Router.HandleFunc("/jokes", h.getJokes).Methods(http.MethodGet)
 	h.Router.HandleFunc("/jokes/add", h.addJoke).Methods(http.MethodPost)
-	h.Router.HandleFunc("/jokes/search?{id}", h.getJokeByID).Methods(http.MethodGet)
-	h.Router.HandleFunc("/jokes/search?{text}", h.getJokesByText).Methods(http.MethodGet)
+	h.Router.HandleFunc("/jokes/id", h.getJokeByID).Methods(http.MethodGet)
```

Comment:
it became worse than it was. You shouldn't introduce a new endpoint for every new parameter in search